File: PHP/1-Foundation/7-Linear-Search/code.php
<?php

function linear_search($arr, $target) {
   for ($i = 0; $i < count($arr); $i++) {

      // If a match is found, return true.
      if ($arr[$i] === $target) {
         return true;
      }
   }

   // No match was found, hence return false.
   return false;
}

File: Python/01-Foundation/6-Linear-Search/code.py
def linear_search(arr, target):
   for item in arr:
      if item == target:
         return True

   return False


# Test the function
print(linear_search([1, 2, 3], 2))
print(linear_search([1, 2, 3], '2'))
print(linear_search(['2', '4', '6'], '2'))
print(linear_search(['2, 6', '1, 4'], '2'))
print(linear_search([False, False, False, True], False))